src/libraries/Microsoft.Extensions.Logging.Abstractions/src/Microsoft.Extensions.Logging.Abstractions.csproj

```diff
@@ -1,8 +1,12 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent);netstandard2.0;net461</TargetFrameworks>
+    <ExcludeCurrentNetCoreAppFromPackage>true</ExcludeCurrentNetCoreAppFromPackage>
     <EnableDefaultItems>true</EnableDefaultItems>
+    <!-- Use targeting pack references instead of granular ones in the project file. -->
+    <DisableImplicitAssemblyReferences>false</DisableImplicitAssemblyReferences>
```

Comment:
Did you intend to check in these changes?

Reply:
yes this was intentional otherwise wouldn't have been able to add NetCoreAppCurrent for the project.

cc: @safern

Reply:
The reason behind adding `NetCoreAppCurrent` for these projects was to have CI protection for the nullable annotations to actually be correct and not broken on by upcoming changes but to actually not ship that asset. However the only caveat that this brings is that by adding this I'm not sure if people can just clone the repo and build this project without pre building anything else which was an effort @ViktorHofer did in the past.

Reply:
> yes this was intentional otherwise wouldn't have been able to add NetCoreAppCurrent for the project I think.

This property is for convenience only, in case you don't want to list all the individual assembly references. In this case I think it's fine as we don't ship the `$(NetCoreAppCurrent)' asset.

> However the only caveat that this brings is that by adding this I'm not sure if people can just clone the repo and build this project without pre building anything

Unfortunately with our current infra that's true but I don't think we should see that as a blocker. I would like to improve our infra so that `$(NetCoreAppCurrent)' is able to build via dependencies without an up-front build which would also enable VS builds for .NETCoreApp without up-front steps.